src/test/kotlin/dev/stalla/model/factory/ItunesCategoryFactoryTest.kt

```diff
@@ -0,0 +1,148 @@
+package dev.stalla.model.factory
+
+import assertk.all
+import assertk.assertThat
+import assertk.assertions.isEqualTo
+import assertk.assertions.isNotNull
+import assertk.assertions.isNull
+import assertk.assertions.prop
+import dev.stalla.model.itunes.ItunesCategory
+import org.junit.jupiter.api.Test
+
+class ItunesCategoryFactoryTest {
+
+    private val validSimpleItunesCategoryNames = listOf(
+        "Arts",
+        "Business",
+        "Comedy",
```

Comment:
Ensure we can retrieve all instances from the factory method, i.e. our manually maintained instance list is complete.